### ISSUE TITLE
Out of sync with au-packages-template

### ISSUE THREAD

**Issue by unsolaci** [chocolatey-community/chocolatey-packages-template#39]
AU scripts seem to be out of date / out of sync with [majkinetor/au-packages-template](https://github.com/majkinetor/au-packages-template).

I can see some sync commits  (57aa601c094cb21c8ce156b84c17b76d24b65cee, 5811f8c101939d89bdd5dc9133ffb699ce042be4, 6738ecbb1e2e5c4d3b25242cbbaffd21a4a13e1e) by @majkinetor over the years, but it seems that there hasn't been one for a while now, even though the AU scripts in [majkinetor/au-packages-template](https://github.com/majkinetor/au-packages-template) are considerably newer.

**Comment by majkinetor** [comment]
Yes, that is the case. I initially handled both, but that is not OK use of my time so I continued only on au-packages-template. Having one template instead of 2 makes much more sense anyway and is less confusing. 

I would deprecate this on but its not up to me.

**Comment by pauby** [comment]
I agree that we should deprecate it and update the documentation.

**Comment by unsolaci** [comment]
> I agree that we should deprecate it and update the documentation.

@pauby what about manual packages though? [majkinetor/au-packages-template](https://github.com/majkinetor/au-packages-template) is only for automatic packages, right?

I know that a directory for manual packages can simply be added to [majkinetor/au-packages-template](https://github.com/majkinetor/au-packages-template) on one's own. But still it would be nice to have a single template for all kinds of packages, that the docs could point to. It's just an opinion though 💡.

**Comment by pauby** [comment]
> majkinetor/au-packages-template is only for automatic packages, right?

It is. We're discussing that template here.

My thoughts were to remove the template entirely and point to the official AU docs.

**Comment by majkinetor** [comment]
No its not.

**Comment by unsolaci** [comment]
> No its not, you can have manual packages. I do have more manual myself then automatic. Its just not separated in different folders. Automatic packages have update.ps1, manual generally don't.

Oh! Okay, so in this case it does indeed seem to make more sense to deprecate and update the docs on that matter.
